e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/SimulationTests.java

```diff
@@ -470,7 +470,9 @@ void directiveIdIncluded() throws IOException {
 
       // The error includes the directive id that was executing
       assertTrue(exception.containsKey("executingDirectiveId"));
+      assertTrue(exception.containsKey("executingActivityType"));
       assertEquals(dirId, exception.getInt("executingDirectiveId"));
+      assertEquals("DaemonCheckerActivity", exception.getString("executingActivityType"));
```

Comment:
`activityStackTrace` needs to be checked here

Reply:
Updated the e2e test to include the missing `actiivtyStackTrace` from this test.